src/routes/file-storage-api/handlers/entitiesHandler.ts

```diff
@@ -149,10 +149,16 @@ const createEntitiesHandler = ({ esClient }: { esClient: Client }): Handler => {
           .boolQuery()
           // TODO: All of the `as string` casting in this section was added to silence the typescript compiler, it needs to be tested
           // the solution is likely in the types being applied to the Request object, such that it doesnt know if the query params are string or parsed into arrays
-          .must([
+          .should([
```

Comment:
why change `must` to `should`?

Reply:
Wasn't supposed to. Thought I changed it back. Nice catch.

Reply:
I'm going to review this again tomorrow... I think the extra query I'm adding is not firing in my test cases, and would break if that filter was provided. I think this extra query filter is not needed and should be removed before merging.